### ISSUE TITLE
multi-organ segmentation 

### ISSUE THREAD

**Issue by lpc-97** [stanfordmlgroup/Auto-Generate-WLs#2]
Hello, your proposed method is very nice. I have a question for you. Is your method applicable to multi-organ segmentation dataset? I look forward to your reply and thank you again for your innovative work.

**Comment by jeya-maria-jose** [comment]
Yes, it is applicable to multi-organ segmentation. You can include the additional organ information as additional channels and the method still would work good.
